### PR TITLE
[#3137] fix(spark-connector):  change table comment though table properties

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
change table comment though table properties

### Why are the changes needed?
Fix: #3137 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
add ut and IT

### CODE REVIEW

File: common/src/main/java/com/datastrato/gravitino/dto/requests/TableUpdateRequest.java [hunk 1]
```diff
@@ -157,9 +157,11 @@ public UpdateTableCommentRequest() {
      */
     @Override
     public void validate() throws IllegalArgumentException {
+      /*
```
Comment (FANNG1) [inline common/src/main/java/com/datastrato/gravitino/dto/requests/TableUpdateRequest.java:160]: Spark supports remove comment though unset table properties `comment`, @jerryshao @mchades  do you think is it reasonable to make `newComment` field in `UpdateTableCommentRequest` null to represent remove comment?

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java [hunk 2]
```diff
@@ -387,6 +388,23 @@ void testAlterTableSetAndRemoveProperty() {
...
+    createSimpleTable(tableName);
+    sql(
+        String.format(
+            "ALTER TABLE %s SET TBLPROPERTIES('%s'='%s')",
+            tableName, ConnectorConstants.COMMENT, comment));
+    SparkTableInfo tableInfo = getTableInfo(tableName);
+    SparkTableInfoChecker checker =
+        SparkTableInfoChecker.create().withName(tableName).withComment(comment);
+    checker.check(tableInfo);
+  }
```
Comment (jerryshao) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java:406]: So Spark uses table property for comment, right?

**Comment by FANNG1** [comment]
@caican00 do you have time to review this PR?

**Review by jerryshao — DISMISSED** [review]
LGTM from my side.

File: spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/BaseCatalog.java [hunk 3]
```diff
@@ -438,10 +438,17 @@ private String getDatabase(NameIdentifier gravitinoIdentifier) {
...
+      if (ConnectorConstants.COMMENT.equals(setProperty.property())) {
+        return com.datastrato.gravitino.rel.TableChange.updateComment(setProperty.value());
+      } else {
+        return com.datastrato.gravitino.rel.TableChange.setProperty(
+            setProperty.property(), setProperty.value());
+      }
     } else if (change instanceof TableChange.RemoveProperty) {
       TableChange.RemoveProperty removeProperty = (TableChange.RemoveProperty) change;
+      Preconditions.checkArgument(
+          ConnectorConstants.COMMENT.equals(removeProperty.property()) == false,
```
[the remark this answers is not in the document]
Comment (FANNG1) [reply spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/catalog/BaseCatalog.java:450]: Gravitino interface doesn't support remove `comment`

**Review by jerryshao — APPROVED** [review]

**Pull request merged**